src/components/Footer/style.ts

```diff
@@ -0,0 +1,116 @@
+import styled from "styled-components";
+
+export const Background = styled.div`
+  background-color: ${(props) => props.theme.colors.fourthUi};
+  width: 100%;
+`;
+
+export const Container = styled.div`
+  max-width: 1800px;
+  max-height: 100%;
+  height: 250px;
```

Comment:
On wider screens footer doesn't look good and it's probably because of that static height value
<img width="1792" alt="Screenshot 2022-08-21 at 12 00 05" src="https://user-images.githubusercontent.com/24607370/185786010-aa5d9464-e369-4710-9723-b469f475ecca.png">